2009-08-09  Yavor Doganov  <yavor@gnu.org>

	* configure.ac: Use AC_SEARCH_LIBS to avoid always appending -lnsl
	to LIBS.


--- gnustep-base.orig/configure.ac
+++ gnustep-base/configure.ac
@@ -2140,7 +2140,7 @@
 # These used by GSFileHandle.m and distributed objects
 # On some systems we need -lnsl ... so check for that first.
 #--------------------------------------------------------------------
-AC_CHECK_LIB(nsl, inet_ntop)
+AC_SEARCH_LIBS([inet_ntop], [nsl])
 AC_CHECK_FUNCS(gethostbyaddr_r inet_aton inet_pton inet_ntop sigaction)
 USE_ZLIB=0
 AC_CHECK_HEADERS(zlib.h)
